admin_manual/release_notes.rst

```diff
@@ -177,6 +177,10 @@ Changes in 9.1
 * Performance and memory usage improvements for Dropbox with stream download
 * GDrive library update provides exponential backoff which will reduce rate limit errors
 
+**Shibboleth**
+
+* The WebDAV endpoint was changed from /remote.php/webdav to /remote.php/dav. You need to check your Apache configuration if you have exceptions or rules for WebDAV configured.
```

Comment:
To make the patch just that much simpler to read, I suggest marking up code samples; in this case: ``/remote.php/webdav`` to ``/remote.php/dav``.